hijack/helpers.py

```diff
@@ -7,7 +7,10 @@
 from django.contrib.auth import login, load_backend, BACKEND_SESSION_KEY
 from django.http import HttpResponseRedirect
 from django.shortcuts import get_object_or_404
-from django.utils.http import is_safe_url
+try:
+    from django.utils.http import url_has_allowed_host_and_scheme
+except ImportError:
+    from django.utils.http import is_safe_url as url_has_allowed_host_and_scheme
```

Comment:
I personally don't really like this approach. And my position is very much in line with this post.
https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

Reply:
Excellent point. This will also make removing abandoned code easier in the future. I'll change it.

Reply:
I am just really used to doing it this way because it's more performant and EAFP is usually encouraged in Python. However, those things don't really apply here, so again, great point.